### PR TITLE
OBJ-149 Update tab titles

### PR DESCRIPTION
**Pull request by mattch1**
Made sure tab title matches the list specified for the MVP on ther ticket:
https://companieshouse.atlassian.net/browse/OBJ-149

### CODE REVIEW

**Review by ICoxCH — APPROVED** [review]

File: views/document-upload.html [hunk 1]
```diff
@@ -1,10 +1,7 @@
 {% extends "layout.html" %}
 
 {% block pageTitle %}
-  {% if errorList.length > 0 %}
-    Error:
-  {% endif %}
-  Object to a company being struck off
+  Add documents to support your objection
```
Comment (markpit) [inline views/document-upload.html:4]: We still need the Error shown - see the govuk guidelines - https://design-system.service.gov.uk/components/error-summary/

Comment (mattch1) [reply views/document-upload.html:4]: Is that just for the two pages I've updated or for all pages?

Comment (markpit) [reply views/document-upload.html:4]: I'm just looking at this PR, I'm assuming the other pages are ok. Looks like it is needed for screen readers when there is a validation error. If there is a Error: condition, i'd leave it in.

Comment (mattch1) [reply views/document-upload.html:7]: Put this back - commit should read "OBJ-149PR added error prefix" not "309"

**Review by markpit — APPROVED** [review]

**Pull request merged**